lms/static/sass/shared/_header.scss

```diff
@@ -19,6 +19,15 @@ header.global {
     min-width: 760px;
   }
 
+  .wrapper-header {
+    @include clearfix();
+    height: 40px;
+    margin: 0 auto;
+    padding: 18px ($baseline/2) 0;
+    max-width: grid-width(12);
```

Comment:
There used to be a top-level `<nav>` that had width styling, which I've replaced with this wrapper.  The wrapper then contains `<nav>` elements that group links together.